C7GameData/Save/TileLocation.cs

```diff
@@ -0,0 +1,17 @@
+namespace C7GameData.Save {
+	public struct TileLocation {
+		public int x, y;
```

Comment:
since vectors now use capitalized X and Y I'm thinking we should follow the same convention

Reply:
#482